0.9/implementation-notes/index.html

```diff
@@ -0,0 +1,923 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Implementation Notes, Oxford Common File Layout Specification</title>
+    <meta charset="utf-8">
+    <meta name="description" content="Client Behaviours, Oxford Common File Layout Specification">
+    <script src="../respec-ocfl.js"
+            async class="remove"></script>
+    <script class="remove">
+        var respecConfig = {
+            specStatus: "unofficial",
+            shortName: "ocfl-spec",
+            includePermalinks: true,
+            editors: [
+                {
+                    name: "Andrew Hankinson",
+                    url: "https://orcid.org/0000-0003-2663-0003",
+                    company: "Bodleian Libraries, University of Oxford",
+                    companyURL: "http://www.bodleian.ox.ac.uk/"
+                },
+                {
+                    name: "Neil Jefferies",
+                    url: "https://orcid.org/0000-0003-3311-3741",
+                    company: "Bodleian Libraries, University of Oxford",
+                    companyURL: "http://www.bodleian.ox.ac.uk/"
+                },
+                {
+                    name: "Rosalyn Metz",
+                    url: "https://orcid.org/0000-0003-3526-2230",
+                    company: "Emory University",
+                    companyURL: "https://web.library.emory.edu/"
+                },
+                {
+                    name: "Julian Morley",
+                    url: "https://orcid.org/0000-0003-4176-1933",
+                    company: "Stanford University",
+                    companyURL: "https://library.stanford.edu/"
+                },
+                {
+                    name: "Simeon Warner",
+                    url: "https://orcid.org/0000-0002-7970-7855",
+                    company: "Cornell University",
+                    companyURL: "https://www.library.cornell.edu/"
+                },
+                {
+                    name: "Andrew Woods",
+                    url: "https://orcid.org/0000-0002-8318-4225",
+                    company: "LYRASIS",
+                    companyURL: "https://lyrasis.org/"
+                }
+            ],
+            edDraftURI: "https://ocfl.io/0.9/implementation-notes/",
+            wg: "Oxford Common File Layout Editors",
+            wgURI: "https://ocfl.io",
+            wgPublicList: "",
+            otherLinks: [{
+                key: "Previous version",
+                data: [
+                    {
+                        value: "N/A"
```

Comment:
This `value` should likely point to [https://ocfl.io/0.3/implementation-notes/](https://ocfl.io/0.3/implementation-notes/)